### PR TITLE
Fix pvcamera

### PR DESCRIPTION
**Pull request by Deedone**
Fixes for pvcamera build to make it work out of the box.

### CODE REVIEW

File: meta-xt-driver-domain/recipes-extended/camerabe/files/camera_be.cfg [hunk 1]
```diff
@@ -19,7 +19,7 @@ mediactl:
 {
     link1 = "'rcar_csi2 feaa0000.csi2':1 -> 'VIN0 output':0 [1]";
     link2 = "'adv748x 4-0070 hdmi':1 -> 'adv748x 4-0070 txa':0 [1]";
-    source_fmt = "'adv748x 4-0070 hdmi':1 [fmt:RGB888_1X24/1920x1080 field:none]";
+    source_fmt = "'adv748x 4-0070 hdmi':1 [fmt:RGB888_1X24/1920x1080 field:none],'adv748x 4-0070 txa':0 [fmt:RGB888_1X24/1920x1080 field:none]";
     sink_fmt = "'rcar_csi2 feaa0000.csi2':1 [fmt:RGB888_1X24/1920x1080 field:none]";
```
Comment (otyshchenko1) [inline meta-xt-driver-domain/recipes-extended/camerabe/files/camera_be.cfg:23]: I am wondering, doesn't it work without that change? Is this suggested elsewhere?

**Comment by otyshchenko1** [comment]
https://github.com/xen-troops/meta-xt-common/pull/47/commits/6a3c7f2fbbe410b0b47f5d7d4157cd41a61e32c4 - lgtm

**Comment by rshym** [comment]
Please fix the commit message: replace 'witch crucial' with 'with critical'.

**Comment by lorc** [comment]
Well, I don't like the idea that `master` branch of camera be is behind latest changes. It is better to align camera_be's master, than to switch branches in this repo

**Pull request closed without merging**